controllers/osbuild_controller.go

```diff
@@ -380,12 +381,14 @@ func (r *OSBuildReconciler) createImageRequest(osBuild *osbuildv1alpha1.OSBuild,
 	return &imageRequest, nil
 }
 
-func (r *OSBuildReconciler) getUploadOptions(osBuild *osbuildv1alpha1.OSBuild, targetImageType osbuildv1alpha1.TargetImageType) (*composer.UploadOptions, error) {
+func (r *OSBuildReconciler) getUploadOptions(osBuild *osbuildv1alpha1.OSBuild, targetImageType osbuildv1alpha1.TargetImageType, logger logr.Logger) (*composer.UploadOptions, error) {
 	var uploadOptions composer.UploadOptions
 	switch uploadTypeForTargetImageType[targetImageType] {
 	case composer.UploadTypesAwsS3:
+		logger.Info("upload to S3")
```

Comment:
Are these logs really add anything? Whoever reads the logs already knows which images go to which target

Reply:
I will remove it...